tests/compat.py

```diff
@@ -34,3 +34,7 @@
     from io import TextIOWrapper
 except ImportError:  # Python < 3
     TextIOWrapper = file
```

Comment:
@jlaska This is the one line that I couldn't could to pass flake8. Here, 'file' is a keyword native to Python... but only for older versions. So I guess it's not surprising that flake8 does not see it as such. But then we're in a bit of a catch 22. How do we keep it functioning correctly and also pass flake8?